tests/conftest.py

```diff
@@ -1390,13 +1389,24 @@ def collect_db_dump_on_duts(request, duthosts):
             db_id = db_config['DATABASES'][db]['id']
             dbs.add(db_id)
 
-        # Collect DB dump
-        duthosts.file(path = db_dump_path, state="directory")
-        for i in dbs:
-            duthosts.shell("redis-dump -d {} -y -o {}/{}".format(i, db_dump_path, i))
+        namespace_list = duthosts[0].get_asic_namespace_list() if duthosts[0].is_multi_asic else ['']
+        if namespace_list != ['']:
+            for namespace in namespace_list:
+                # Collect DB dump
+                db_dump_path = os.path.join(dut_file_path + "/" + namespace, request.module.__name__, request.node.name)
+                duthosts.file(path = db_dump_path, state="directory")
+                for i in dbs:
```

Comment:
I think we skip `STATEDB` on 201911 because of issue https://github.com/Azure/sonic-buildimage/issues/5587
There is a fix (https://github.com/Azure/sonic-buildimage/pull/5646) for that issue but somehow it was not included in 201911 image.

Reply:
OK, I skipped it below